packages/core/src/app/customer/Customer.spec.tsx

```diff
@@ -107,11 +107,12 @@ describe('Customer', () => {
             await new Promise(resolve => process.nextTick(resolve));
             component.update();
 
-            expect(component.find(StripeGuestForm).exists()).toEqual(true);
```

Comment:
`Prettier` modified a lot of lines after running `npm run lint`.
Put them into the PR as these lines would be modified anyway.